src/compiler/emitter.ts

```diff
@@ -491,20 +497,120 @@ module ts {
                     }
                 }
 
+                function convertToBase64(input: string): string {

```

Comment:
I'vegot some extremely simple code to do this here:
https://code.google.com/p/metasyntactic/source/browse/trunk/MetasyntacticShared/Classes/Base64.m?r=4345

Encoding is only 20 lines.